Configuration/Geometry/python/GeometryExtended2023Reco_cff.py

```diff
@@ -15,13 +15,12 @@
 #Muon
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *
-from Geometry.GEMGeometry.gemGeometry_cfi import *
```

Comment:
Hi Yana, 

This file was indeed deleted. Could you instead add the following lines:
<PRE>
from Geometry.GEMGeometryBuilder.gemGeometry_cfi import *
from Geometry.GEMGeometryBuilder.me0Geometry_cfi import *
</PRE>